src/utils/EquipmentType.js

```diff
@@ -205,19 +220,18 @@ export const renderEquipmentForSearchBar = (classes, intl) => {
                                 {part.text}
                             </span>
                         ))}
-                    </span>
-                    {element.type !== EQUIPMENT_TYPE.SUBSTATION.name &&
-                        element.type !== EQUIPMENT_TYPE.VOLTAGE_LEVEL.name && (
-                            <span
-                                className={clsx(
-                                    classes.equipmentTag,
-                                    classes.equipmentVlTag
-                                )}
-                            >
-                                {element.voltageLevelLabel}
-                            </span>
-                        )}
+                    </OverflowableText>
                 </div>
+                {element.type !== EQUIPMENT_TYPE.SUBSTATION.name &&
```

Comment:
You don't need a parent `div` here with `style={{overflow: 'hidden'}}` ?
This is because you have a `equipmentVlTag` class which set a static width, then the {{overflow: 'hidden'}} inside OverflowableText works ?

Reply:
Yes you can remove the div element

Reply:
done